lmn/templates/lmn/notes/new_note.html

```diff
@@ -50,6 +50,9 @@ <h2>New note for {{ show.artist.name }} at {{ show.venue.name }} on {{ show.show
           <br>
           5 - Very Satisfied
           </br>
+          <br>
+          {{ form.rating.errors }}
```

Comment:
Works now! Is there a way to highlight this or move this on top of the form so is easily visible?